### PR TITLE
measure duration ms

### PR DESCRIPTION
**Pull request by josler**

### CODE REVIEW

File: monitoring/sentry_monitor.go [hunk 1]
```diff
@@ -11,15 +11,15 @@ type SentryMonitor struct {
...
-	client, err := raven.NewClient(ravenDSN, nil)
+	client, err := raven.New(ravenDSN)
 	if err != nil {
 		return nil, err
 	}
 	return &SentryMonitor{ravenClient: client}, nil
 }
 
 func (sm *SentryMonitor) CaptureException(err error) {
-	sm.ravenClient.CaptureErrorAndWait(err, map[string]string{}, raven.NewStacktrace(2, 3, nil))
```
Comment (mariokostelac) [inline monitoring/sentry_monitor.go]: CaptureError is async?

File: metrics/statsd_recorder.go [hunk 2]
```diff
@@ -40,6 +40,10 @@ func (m *StatsdRecorder) MeasureSince(metricName string, since time.Time) {
 	m.Metrics.MeasureSince(m.prefixedMetricName(metricName), since)
 }
 
+func (m *StatsdRecorder) MeasureDurationMS(metricName string, durationMS float32) {
+	m.Metrics.AddSample(m.prefixedMetricName(metricName), durationMS)
```
Comment (mariokostelac) [inline metrics/statsd_recorder.go]: why we have statsd_recorder and datadog_statsd_recorder?

Comment (josler) [inline metrics/statsd_recorder.go]: datadog statsd adds tags, which raw statsd does not.

File: monitoring/sentry_monitor.go (hunk 1 above)
Comment (josler) [inline monitoring/sentry_monitor.go]: ya

**Pull request merged**